### ISSUE TITLE
print statements

### ISSUE THREAD

**Issue by tischi** [haesleinhuepf/napari-script-editor#7]
@haesleinhuepf Where do the print statements go? I was expecting the terminal window, but I cannot see them there.

**Comment by haesleinhuepf** [comment]
Hi Christian @tischi ,

I see the output in the terminal:

![image](https://user-images.githubusercontent.com/12660498/152394920-d834fa09-0405-4bcf-92af-4860df1d6748.png)

Can you confirm?

Best,
Robert

**Comment by tischi** [comment]
OK, yes, I wasn't looking there!

**Comment by haesleinhuepf** [comment]
I'm reopening this, because I think it would be cool to have a text-output window just below the text editor window. It could look like this:

![image](https://user-images.githubusercontent.com/12660498/152403637-2835c467-8619-4f3c-8d3b-75fe7e90f484.png)

**Comment by tischi** [comment]
Yes, exactly, for non-experts (like me) finding the output in the terminal window where napari has been launched is not so easy 😺
